doc/classes/Input.xml

```diff
@@ -47,8 +47,9 @@
 			<return type="Vector3">
 			</return>
 			<description>
-				If the device has an accelerometer, this will return the acceleration. Otherwise, it returns an empty [Vector3].
+				If the device has an accelerometer, this will return the acceleration. Otherwise, it will return [constant Vector3.ZERO].
 				Note this method returns an empty [Vector3] when running from the editor even when your device has an accelerometer. You must export your project to a supported device to read values from the accelerometer.
+				[b]Note:[/b] This method is only implemented on iOS, Android and UWP. On other platforms, it will always return [constant Vector3.ZERO].
```

Comment:
Minor tweaks, like adding the oxford comma (comma before the last element of an enumeration, see docs writing guidelines).

```suggestion
				[b]Note:[/b] this method only works on iOS, Android, and UWP. On other platforms, it always returns [constant Vector3.ZERO].
```

Reply:
For notes, I prefer always capitalizing the sentence as if "Note:" isn't part of the sentence. I've always done it this way and it seems more logical to me.